.githooks/pre-push

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
`shellcheck` says

```
In prova.sh line 11:
  find Cabal Cabal-syntax cabal-install -name '*.hs' \
  ^-- SC2038: Use -print0/-0 or -exec + to allow for non-alphanumeric filenames.


In prova.sh line 17:
    | xargs -P ${PROCS} -I {} fourmolu -q --mode check {}
               ^------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
    | xargs -P "${PROCS}" -I {} fourmolu -q --mode check {}

For more information:
  https://www.shellcheck.net/wiki/SC2038 -- Use -print0/-0 or -exec + to allo...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

Reply:
maybe (not sure) `set -e` too?